sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample10_SecretReference.md

```diff
@@ -0,0 +1,50 @@
+# Create, Retrieve and Delete a Secret Reference
+
+The AppConfiguration service supports KeyVault Secret reference settings.
+Secret references are settings that follow specific JSON schema for the `Value`, and the `application/vnd.microsoft.appconfig.keyvaultref+json;charset=utf-8` content type.
+The `Azure.Data.AppConfiguration` library provides a strongly-typed way of managing Secret references.
+This sample shows how to use the library to create, retrieve, and delete Secret references.
+
+## Create a Secret Reference
+
+To create a secret reference, use the `SecretReferenceConfigurationSetting` class:
+
+```C# Snippet:Sample_CreateSecretReference
+var secretReferenceSetting = new SecretReferenceConfigurationSetting("setting", new Uri("https://<keyvault_name>.vault.azure.net/secrets/<secret_name>"));
```

Comment:
is the Uri a value the user can get in any way? 
I worry if the user wants to use another cloud, that they might think that the format of the Uri is always this and will miss that the DNS suffix could vary

Reply:
Yeah, both via the portal and the KV SDK.

Reply:
Consider adding a comment so users can know where to get that value from